tests/environment.yml

```diff
@@ -1,77 +1,80 @@
-name: stko-test
+name: stko_tests
```

Comment:
Will changing the version of these dependencies affect the existing tests?

Reply:
No, because the tests are run based on this environment, every time. So if they do, then the tests will fail.